src/math/float-packing.js

```diff
@@ -5,16 +5,25 @@ const oneDiv255 = 1 / 255;
 const floatView = new Float32Array(1);
 const int32View = new Int32Array(floatView.buffer);
 
-// utility static class providing functionality to pack float values to various storage representations
+/**
+ * Utility static class providing functionality to pack float values to various storage
+ * representations.
+ *
+ * @ignore
```

Comment:
just curious if we need ignore tag on modules that are not exported from index.js?

Reply:
We do, yes, because JSDoc doesn't know about import/export statements and can't discern public from internal API. You have to explicitly use the @ignore everywhere where it's necessary.

Reply:
As time passes, it seems JSDoc is getting smarter though and leveraging the code to minimize the amount of tag usage. Today, you still need to use `@augments`/`@extends` but I suspect they make that unnecessary at some point: https://github.com/jsdoc/jsdoc/issues/1843